src/LandIce/evaluators/LandIce_LiquidWaterFraction.hpp

```diff
@@ -26,13 +26,13 @@ namespace LandIce
   This evaluator computes the liquid water fraction in temperate ice
  */
 
-template<typename EvalT, typename Traits, typename Type>
+template<typename EvalT, typename Traits, typename EntalpyType>
```

Comment:
Minor: `Entalpy`->`Enthalpy`. Also, we could be more verbose, and use `EnthalpyST` or `EnthalpyScalarT`, to emphasizes the template arg is the _scalar type_ of the enthalpy (not the "type of enthalpy", which I'm not sure what would mean). Also, `BlahST` is somewhat the common name used in other evaluators for the scalar type of field Blah.